apps/server/src/modules/board/controller/dto/element/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from './any-content-element.response';
-export * from './create-content-element.body';
-export * from './element-content-update.body.params';
+export * from './create-content-element.body.params';
+export * from './update-element-content.body.params';

```

Comment:
Rename to **update-content-element.body.params**

Reply:
I think so too. But Thomas said there is a diffence between update and create and the namings are correct as it is here. One thing I would like to discuss.

Reply:
Okay